### PR TITLE
Complete the chainpal website

### PR DESCRIPTION
**Pull request by KBhautik6**
Part 1
Screenshot 2023-12-30 at 10 01 13 AM
Part 2
Screenshot 2023-12-30 at 10 01 24 AM

Zoom out
Screenshot 2023-12-30 at 10 01 34 AM

Responsive section-2
Screenshot 2023-12-30 at 10 02 02 AM
Responsive section -2
Screenshot 2023-12-30 at 10 02 12 AM

 - [x] Checking the code is duplicate or not?
 - [x] Is these code has bug
 - [x] Is code is check by ankit sir?
 - [ ] Are you going to merge this or not?

### CODE REVIEW

File: Image/Document.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<!-- saved from url=(0022)http://127.0.0.1:5500/ -->
```
Comment (ankitadrixus08) [inline Image/Document.html]: remove unwanted code @KBhautik6

File: Image/Document.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ankitadrixus08) [inline Image/Document.html]: use proper name "chainpals" here @KBhautik6

File: Image/Document.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    
+<link href="./Document_files/style.css" rel="stylesheet"></head>
```
Comment (ankitadrixus08) [inline Image/Document.html]: link is wrong @KBhautik6

Comment (ankitadrixus08) [inline Image/Document.html]: use proper link here this is not working here @KBhautik6

File: Image/Document.html [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+<body>
+    <main class="maincontainer">
+        <div class="section">
+            <section class="photosection">
+                <div class="escrowdiv">
+                    <div class="wraping">
+                        <div class="leftdiv">
+                            <img src="./Document_files/secure.png" alt="Become" class="imagecontent">
```
Comment (ankitadrixus08) [inline Image/Document.html]: same for all files and links please make it proper like images came from "images" folder only @KBhautik6

File: Image/Document.html [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+                        </div>
+                        <div class="rightdiv">
+                            <h3 class="heading">Join the Escrow Manager Program</h3>
+                            <p class="description">Become a Chainpals Escrow Manager and start earning passive income by hodling EM NFT.</p>
+                            <div class="wraperoflearnmore">
+                                <a href="https://www.chainpals.io/partner" class="a-learnmore">LEARN MORE</a>
```
Comment (ankitadrixus08) [inline Image/Document.html]: not needed a- here @KBhautik6

File: Image/Document.html [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                
+            </section>
+            <section class="roadmap">
+                <div class="maindiv2">
```
Comment (ankitadrixus08) [inline Image/Document.html]: change class name @KBhautik6

File: Image/Document.html [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+                    <div class="backgroundimg">
```
Comment (ankitadrixus08) [inline Image/Document.html]: why this here use before for shield

File: Image/Document.html [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+                        <div class="roadmapwrapper margin-top-40">
+                            <div class="wrapper"> 
+                                <!-- idhare se change hota ha vo upara wali line wraper ha niche wali roadmapswiper -->
+                                <!-- Q4 2021-->
+                                <div class="roadmapswiper  text-align-centre">
+                                    <div class="timestampdiv flexapply">
+                                        <span class="date">Q4 2021</span>
+                                    </div>
+                                    <div class="line-dot">
+                                        <div class="dot"></div>
```
Comment (ankitadrixus08) [inline Image/Document.html]: same blank div is not acceptable @KBhautik6

File: Image/Document.html [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                    </div>
+                </div>
+            </div></section>
+        </div>
+    </main>
+<!-- Code injected by live-server -->
+<script>
```
Comment (ankitadrixus08) [inline Image/Document.html]: can you explain this code @KBhautik6

File: Image/Document_files/style.css [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+}
+.timestampdiv{
+    color: white;
+    width: 100%;
+    margin-bottom: 20px;
+    flex-direction: column;
+    
+}
+.date{
+    font-size:100%;
```
Comment (ankitadrixus08) [inline Image/Document_files/style.css]: why 100% here @KBhautik6 what it will take

File: Image/Document_files/style.css [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+}
+.statuswrapper::before .flexapply::before{
+    content: " ";
+    position: absolute;
+    width: 100%;
+    height: 23px;
+    background-color:#0749f8;
+    border-radius: 0;
+    top: -8px;
+    /* margin:20px */
```
Comment (ankitadrixus08) [inline Image/Document_files/style.css]: remove unwanted code @KBhautik6

File: Image/Document_files/style.css [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+    background-color: blue;
+    top: -8px;
+    margin: 0 auto;
+}
+.statuswrapper .status::before{
+    content: "";
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    background-color: #fff;
```
Comment (ankitadrixus08) [inline Image/Document_files/style.css]: use same everywhere please use hexacode @KBhautik6

File: Image/Document_files/style.css [hunk 13]
```diff
@@ -0,0 +1,245 @@
...
+    border-radius: 25px;
+    top: -35px;
+    left:10px;
+    margin: 0 auto;
+}
+/* .line-dot {
```
Comment (ankitadrixus08) [inline Image/Document_files/style.css]: remove unwated code @KBhautik6

Comment (KBhautik6) [reply Image/Document_files/style.css]: Okay sir

File: index.html [hunk 14]
```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ankitadrixus08) [inline index.html]: Document >> chainpals please change

File: index.html [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+    <link href="./style.css" rel="stylesheet">
+</head>
+<body>
+    <main class="maincontainer">
+        <div class="section">
+            <section class="photo-section">
+                <div class="escrow-div">
+                        <h3 class="heading-responsive">Join the Escrow Manager Program</h3>
```
Comment (ankitadrixus08) [inline index.html]: what is the meaning of responsive in class @KBhautik6

File: index.html [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+                    <div class="wraping">
```
Comment (ankitadrixus08) [inline index.html]: wraping .> > wrapping

File: index.html [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+                            <p class="description">Become a Chainpals Escrow Manager and start earning passive income by hodling EM NFT.</p>
+                            <div class="wraper-of-learn-more">
+                                <a href="https://www.chainpals.io/partner"  class="a-learnmore">LEARN MORE</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                
+            </section>
+            <section class="roadmap">
```
Comment (ankitadrixus08) [inline index.html]: roadmap >> roadmapSection

File: index.html [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+                        <h3 class="heading-responsive">Join the Escrow Manager Program</h3>
+                    <div class="wraping">
+                        <div class="left-div">
+                            <img src="./Images/become-partner.png" alt="Become" class="image-content">
+                        </div>
+                        <div class="right-div">
+                            <h3 class="heading">Join the Escrow Manager Program</h3>
+                            <p class="description">Become a Chainpals Escrow Manager and start earning passive income by hodling EM NFT.</p>
+                            <div class="wraper-of-learn-more">
+                                <a href="https://www.chainpals.io/partner"  class="a-learnmore">LEARN MORE</a>
```
Comment (ankitadrixus08) [inline index.html]: a-learnmore >> learnMoreBtn

File: index.html [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                
+            </section>
+            <section class="roadmap">
+                <div class="roadmap-div">
+                    <div class="background-img">
```
Comment (ankitadrixus08) [inline index.html]: dont need blank class here use that bg in backgroung @KBhautik6

File: index.html [hunk 20]
```diff
@@ -0,0 +1,148 @@
...
+                    </div>
+                    <div class="main-context">
+                        <h2 class="heading-roadmap text-align-centre">Road Map</h2>
+                        <div class="road-map-wrapper margin-top-40">
+                            <div class="wrapper"> 
+                                <!-- idhare se change hota ha vo upara wali line wraper ha niche wali road-map-swiper -->
+                                <!-- Q4 2021-->
+                                <div class="road-map-swiper  text-align-centre">
+                                    <div class="times-stamp-div flexapply">
+                                        <span class="date">Q4 2021</span>
```
Comment (ankitadrixus08) [inline index.html:44]: flexapply >> flex

File: index.html [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+                                    </div>
+                                    <div class="status-wrapper">
+                                        <div class="status">
```
Comment (ankitadrixus08) [inline index.html]: status >> stausBox

File: style.css [hunk 22]
```diff
@@ -0,0 +1,338 @@
...
+    text-decoration: none;
+}
+
+.roadmap{
+    position: relative;
+    padding: 120px 0 50px ;
+}
+.background-img{
+    position: absolute;
+    background-image: url(./Images/roadmap.svg);
```
Comment (ankitadrixus08) [inline style.css]: use shorthand property here @KBhautik6

File: style.css [hunk 23]
```diff
@@ -0,0 +1,338 @@
...
+    font-size: 18px;
+}
+.time-stamp-div{
+    color: #fff;
+    width: 100%;
+    margin-bottom: 20px;
+    flex-direction: column;
+    
+}
+.date{
```
Comment (ankitadrixus08) [inline style.css:150]: don't use preserved tags name in class @KBhautik6

File: style.css [hunk 24]
```diff
@@ -0,0 +1,338 @@
...
+.flexapply{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.status-wrapper::before {
+    content: "";
+    border-radius: 25px ;
+    position: absolute;
+    width: 109%;
```
Comment (ankitadrixus08) [inline style.css:166]: why 109% here @KBhautik6

File: style.css [hunk 25]
```diff
@@ -0,0 +1,338 @@
...
+    height: 20px;
+    background-color: #1717f7;
+    top: -8px;
```
Comment (ankitadrixus08) [inline style.css:169]: dont use minus here @KBhautik6

File: style.css [hunk 26]
```diff
@@ -0,0 +1,338 @@
...
+    letter-spacing: normal;
+    color:#bdc2cb ;
+}
+.status::before{
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    background-color:#ffffff;
+    border-radius: 25px;
+    top: -35px;
```
Comment (ankitadrixus08) [inline style.css:203]: same as above @KBhautik6

File: style.css [hunk 27]
```diff
@@ -0,0 +1,338 @@
...
+    .heading{
+        display: none;
+    }
+    body{
+        overflow: auto;
+    }
+    .photo-section{
+        margin: 50px 0;
+    }
+    .heading-responsive{
```
Comment (ankitadrixus08) [inline style.css]: why you need this whole css here is it not possible to do other way @KBhautik6

File: style.css [hunk 28]
```diff
@@ -0,0 +1,338 @@
...
+        line-height: 20px;
+        margin: 0 0 15px 0;
+    }
+    .a-learnmore{
+        padding: 9px 15px;
+        font-size: 12px;
+    }
+
+    .background-img{
+        background-image: url("./Images/roadmapresponse.svg");
```
Comment (ankitadrixus08) [inline style.css]: use shorthand prop @KBhautik6

File: style.css [hunk 29]
```diff
@@ -0,0 +1,338 @@
...
+        position: relative;
+    }
+    .status{
+        text-align: left;
+        font-size: 14px;
+        line-height: 18px;
+        padding: 10px;
+        top: 0;
+    }
+    .date-responsive{
```
Comment (ankitadrixus08) [inline style.css]: same why use 2 times this css @KBhautik6

**Review by ankitadrixus08 — COMMENTED** [review]
18m

**Review by ankitadrixus08 — APPROVED** [review]
Nice work please be careful for naming next time

12m

**Pull request merged**